### DISCUSSION TITLE
Let players sign the maps and stop other players copying the map

### DISCUSSION THREAD

**Discussion by siulung201314** [Zrips/CMI#6346] [Feature Requests]
Some players in my server has made a lot of art maps and sell the art maps to other players.
However, some bad players copy the maps by using blank maps to resell the maps.
I hope that CMI has a feature that players can sign the maps and only the author can copy the signed maps.
The name of author will be shown in the lore of map.
The authors can unsign their own signed map if they want.
If the author changes his player names, his name in the lore of the map also changes.

**Comment by ghost** [comment]
https://www.spigotmc.org/resources/signit.94720/
You can try this plugin
